client/client_test.go

```diff
@@ -140,8 +138,8 @@ func TestClientRetryNonTxn(t *testing.T) {
 	// intent to be pushed and once with priorities which will not.
 	for i, test := range testCases {
 		key := proto.Key(fmt.Sprintf("key-%d", i))
-		txnPri := 1
-		clientPri := 1
+		var txnPri int32 = 1
```

Comment:
I prefer the syntax `txnPri := int32(1)` if you need an explicit type for `txnPri`. Doesn't particularly matter, though.